postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json

```diff
@@ -3274,6 +3274,72 @@
 					},
 					"response": []
 				},
+				{
+					"name": "Plaid Sameday Auth",
+					"protocolProfileBehavior": {
+						"disabledSystemHeaders": {
+							"content-type": true
+						}
+					},
+					"request": {
+						"auth": {
+							"type": "apikey",
+							"apikey": [
+								{
+									"key": "value",
+									"value": "private-key; Authsignature={{app_private_key}}; Usersignature={{user_private_key}}",
+									"type": "string"
+								},
+								{
+									"key": "key",
+									"value": "Authorization",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [
+							{
+								"key": "label",
+								"value": "",
+								"type": "text"
+							}
+						],
```

Comment:
I think you'll still need to put a forwarding header in here to tell the signerserver to forward to remote host

Reply:
Hey @freemish, I ended up duplicating another endpoint and updated the values to keep the headers consistent and verified the endpoint is working.  Let me know if new proposed changes are closer to what you were looking for.  Thanks!